assets/js/util/date-range.js

```diff
@@ -86,25 +64,31 @@ export function getCurrentDateRangeSlug() {
  * @return {Object} The object hash where every key is a date range slug, and the value is an object with the date range slug and its translation.
  */
 export function getAvailableDateRanges() {
-	/* translators: %s: Number of days to request data. */
-	const format = __( 'Last %s days', 'google-site-kit' );
+	const label = ( days ) => sprintf(
+		/* translators: %s: Number of days to request data. */
```

Comment:
See my above comment - here the translator comment is more accurate than you have it elsewhere, but maybe a bit _too_ specific. Let's use `translators: %s: number of days` throughout.